packages/react-components/src/components/ChatMessage/MentionSuggestionRenderer.tsx

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+import React from 'react';
+import { Mention } from '../MentionPopover';
+
+/**
+ * Provides the default implementation for rendering an Mention
+ * @param suggestion - The suggestion to render
+ *
+ * @beta
+ */
+export const defaultMentionSuggestionRenderer = (suggestion: Mention): JSX.Element => {
+  return (
+    <msft-mention id={suggestion.id} displayText={suggestion.displayText}>
```

Comment:
Would you provide an example code of how contoso use this custom html tag? Is it by using some kinda document.findElemetByTag or something?

Reply:
I guess findElementById would be another way, but need a real example to understand this part better